media-table-schema.json

```diff
@@ -69,6 +69,15 @@
       },
       "example": "https://multimedia.agouti.eu/assets/6d65f3e4-4770-407b-b2bf-878983bf9872/file"
     },
+    {
+      "name": "public",
```

Comment:
@peterdesmet I like your suggestion to rename this field to `filePathPublic` - it is more self-explanatory than just `public`

no more comments on this

Reply:
I know I suggested the term, but `filePathPublic` makes me (and @PietrH) intuitively wonder if only the _path_ or also the _file_ are public. So we'd suggest `filePublic` (`TRUE/FALSE`) from all these alternatives:

- `public`
- `filePublic`
- `publicFile` (odd one in all the the `fileX` term names)
- `filePathPublic` (only the path?)
- `fileAccessible` (publicly though?)
- `fileAvailable` (publicly though?)
- `filePubliclyAccessible` (a bit long)
- `filePubliclyAvailable` (a bit long)

Does that work for you?